### PR TITLE
Test: Mask unit test warning

### PR DESCRIPTION
**Pull request by alfred2g**
Mask unit test warning

Description
-----------
Jenkins was failing the unit test because of a warning, removing the warning as it is previously tested

Test Steps
-----------
Unit Tests pass

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: FreeRTOS/Test/CMock/queue/generic/queue_create_dynamic_utest.c [hunk 1]
```diff
@@ -38,6 +38,7 @@
 #include "queue.h"
 
 /* ============================  GLOBAL VARIABLES =========================== */
+extern bool xMaskAssertAndAbort;
```
Comment (paulbartell) [inline FreeRTOS/Test/CMock/queue/generic/queue_create_dynamic_utest.c:41]: I'm not sure I understand why this needs to be moved around.

Comment (paulbartell) [reply FreeRTOS/Test/CMock/queue/generic/queue_create_dynamic_utest.c:41]: Get/Set functions would be cleaner if declaring this in the header file is problematic.

Comment (alfred2g) [reply FreeRTOS/Test/CMock/queue/generic/queue_create_dynamic_utest.c:41]: we can't declare variables inside header files with multiple translation units
the linker will be confused with multiple declarations.
Yeah +1 for Set/Get functions

**Review by paulbartell — APPROVED** [review]

**Pull request merged**